projects/opendr_ws/README.md

```diff
@@ -2,65 +2,96 @@
 
 ## Description
 This ROS workspace contains ROS nodes and tools developed by OpenDR project. Currently, ROS nodes are compatible with ROS Noetic.
-This workspace contains the `ros_bridge` package, which provides message definitions for ROS-compatible OpenDR data types,
+This workspace contains the `opendr_bridge` package, which provides message definitions for ROS-compatible OpenDR data types,
 as well the `ROSBridge` class which provides an interface to convert OpenDR data types and targets into ROS-compatible
-ones similar to CvBridge. You can find more information in the corresponding [documentation](../../docs/reference/rosbridge.md).
+ones similar to CvBridge. You can find more information in the corresponding [documentation](../../docs/reference/opendr-ros-bridge.md).
 
 
-## Setup
-For running a minimal working example you can follow the instructions below:
+## First time setup
+For the initial setup you can follow the instructions below:
 
-0. Source the necessary distribution tools:
+0. Make sure ROS noetic is installed: http://wiki.ros.org/noetic/Installation/Ubuntu (desktop full install)
 
-   ```source /opt/ros/noetic/setup.bash```
+1. Open a new terminal window and source the necessary distribution tools:
+    ```shell
+    source /opt/ros/noetic/setup.bash
+    ```
+   _For convenience, you can add this line to your `.bashrc` so you don't have to source the tools each time you open a  terminal window._
+2. Install the following dependencies, required in order to use the OpenDR ROS tools:
+    ```shell
+    sudo apt-get install ros-noetic-vision-msgs ros-noetic-geometry-msgs ros-noetic-sensor-msgs ros-noetic-audio-common-msgs
```

Comment:
Yes you are right. Should i add it in the installation script in this branch/PR or should we handle it in a separate one?

Reply:
I think we can do the change in this PR, that way we avoid synchroinizing yet another PR whenever something gets merged into develop, since there's plently of open PRs already